app/src/main/java/com/swent/assos/model/navigation/HomeNavigation.kt

```diff
@@ -40,19 +42,23 @@ fun HomeNavigation(navigationActions: NavigationActions) {
     HorizontalPager(
         state = pagerState,
         userScrollEnabled = false,
-        modifier = Modifier.fillMaxSize().weight(1f),
+        modifier = Modifier
+            .fillMaxSize()
+            .weight(1f),
     ) { page ->
       when (page) {
         0 -> News()
         1 -> Overview(navigationActions = navigationActions)
         2 -> Calendar()
-        3 -> QrCode()
+        3 -> FollowAssociation() //QrCode()
```

Comment:
I believe you changed this line to test your code. Please put the previous line back in